action.yaml

```diff
@@ -18,160 +18,187 @@ inputs:
-            # If we do have the base ref, it can be a branch name, so we want to
-            # parse it. This will be a pass through for a SHA
-            FROM="$(git show-ref --hash "$FROM" | tail -n1)"
-          fi
-        fi
-        # If we don't have a base ref, we try to find the history to the default
-        # branch, e.g. main, which should be a sane check.
-        if [[ -z "$FROM" ]]; then
-          # This finds the default branch name from the remote information, e.g. "main", and then parses it to a SHA
-          FROM="$(git show-ref --hash "$(git remote show $(git remote -v | grep push | awk '{print $2}') | grep 'HEAD branch' | awk '{print $3}')" | tail -n1)"
-          echo "::notice::Could not find base ref, trying to use default branch"
-          if ! git merge-base --is-ancestor "$FROM" HEAD; then
-            echo "::warning::Default branch is not an ancestor of HEAD"
-            FROM=""
-          fi
-        fi
-        # If the pull_request event metadata doesn't exist, try to grab the
-        # "before" commit from the push event metadata. This will usually only
-        # give us a single commit to check, but it'll be the latest one.
-        if [[ -z "$FROM" ]]; then
-          # This is a SHA so no parsing needed
-          FROM="${{ github.event.before }}"
-          echo "::warning::Could not find ancestor ref, falling back"
-          if ! git merge-base --is-ancestor "$FROM" HEAD; then
-            echo "::warning::Before commit $FROM is not an ancestor of HEAD"
-          else
-            echo "::notice::Checking commits since $FROM (usually the latest commit)."
-          fi
-        fi
-        # Default to looking at the last 20 commits otherwise.
-        if [[ -z "$FROM" ]]; then
-          FROM="HEAD~20"
-          echo "::warning::Could not find commit range for commitlint"
-          echo "::notice::Checking 20 most recent commit messages."
-        fi
-        commitlint --config "${{ inputs.config-file }}" \
-          --verbose \
-          --color \
-          --from "$FROM" \
-          --to HEAD
-    - name: Restore commitlint config
-      if: always() && hashFiles(inputs.config-file) != '' && inputs.turo-conventional-commit == 'true'
-      shell: bash
-      run: git checkout -- "${{ inputs.config-file }}" || true
-    # Collect file changes
-    - name: Find changed files
-      if: inputs.only-changed == 'true'
-      uses: trilom/file-changes-action@v1.2.4
-      id: file_changes
+      # TODO: use a released version of this action
+      uses: open-turo/action-pre-commit/commitlint@conditionalize-config
       with:
-        output: " "
-    # Export changed file names for use in pre-commit action
-    - name: Limit pre-commit to changed files
+        restore-config: "false"
+        config-file: ${{ inputs.config-file }}
+        turo-conventional-commit: ${{ inputs.turo-conventional-commit }}
+    # - name: Install @commitlint/cli
```

Comment:
we could remove the comment block

Reply:
Yeap wanted to make sure there wasn't an issue running the sub-action beforehand.